.github/ISSUE_TEMPLATE/sub-task.md

```diff
@@ -1,6 +1,6 @@
 ---
-name: 🧩  Unestimated sub-task
-about: "Specify an unestimated sub-task. (Avoid comments.  Use only as prescribed.)"
+name: 🧩 Sub-task
```

Comment:
~~note space character change (I dunno if it was a non-breaking space-- meaning `option+spacebar` or just two spaces, but some of those are there to make the emojis line up)~~   make sure the emojis still line up